### ISSUE TITLE
Wave scripts should not redirect standard output away from the log file

### ISSUE THREAD

**Issue by WalterKolczynski-NOAA** [NOAA-EMC/global-workflow#296]
Jen Yang 2021-03-12 06:53:53 UTC
The gfs wave jobs run always redirect the standard output to files in working directory, therefore a lot of job execution is missing in the standard job output, when the job finishes the working directory is removed. All the log files are gone, this make it very hard for debug or trace the job input and output. Please improve the job output logging to include the execution trace in the job output.

This can be fixed in the next upgrade.

Refs: [Bugzilla #1219](http://www2.spa.ncep.noaa.gov/bugzilla/show_bug.cgi?id=1219)

**Comment by aerorahul** [comment]
I don't think this should be closed, because there are plenty of the other wave jobs (wavepostpnt, wavebndpnt, wavebndpntbll, etc) that do not redirect stdout away from the log file.
Reopening issue!

**Comment by JessicaMeixner-NOAA** [comment]
@aerorahul  - I just checked log files and I think everything - but possibly wave_prep now puts the output to the log files instead of redirecting. 

I think this issue can now be closed.

**Comment by aerorahul** [comment]
Thanks @JessicaMeixner-NOAA
